Library/Homebrew/formula.rb

```diff
@@ -1695,6 +1702,15 @@ class << self
     # <pre>revision 1</pre>
     attr_rw :revision
 
+    # @!attribute [w] revision
+    # Used for creating new Homebrew versions shemes. For example, if we want
+    # to change version scheme from one to another, then we may need to update
+    # `epoch` of this {Formula} to be able to use new version scheme.
```

Comment:
Might want to note explicitly the e.g. moving from 20151020 scheme to 1.0.0 and note that, without this, the prior scheme will always equate to a higher version.